.gitignore

```diff
@@ -30,3 +30,5 @@ core
 /.project
 /tags
 /TAGS
+.vscode/settings.json
+.vscode/c_cpp_properties.json
```

Comment:
I don't think that editor specific files belong to the top level .gitignore.
You can ignore them locally.
@patacongo what's your take?